src/main/java/models/User.java

```diff
@@ -0,0 +1,77 @@
+package models;
+
+import java.util.Objects;
+
+public class User {
+    private String name;
+    private String nickName;
+    private Integer id;
+    private Integer gameCount = 0;
```

Comment:
Why do we need to use the wrapper on int here?

Reply:
- [x] change Integer type to int type for gameCount field